backend/app_1/models.py

```diff
@@ -46,9 +46,20 @@ def __str__(self):
         return f"{self.container} - {self.product}"
     
 class ContainerMass(models.Model):
-    container = models.ForeignKey(Container, on_delete=models.PROTECT)
+    class MealChoices(models.TextChoices):
```

Comment:
It's up to you, but you can also define this class outside `ContainerMass`. If it's defined inside, then the only way to access it is through `ContainerMass`

Reply:
Halo, was denkst du?